src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -21,7 +24,7 @@ object EasyStageDataset {
 
     implicit val s = Settings(
       ownerId = props.getString("owner"),
-      submissionTimestamp = conf.submissionTimestamp().toString,
+      submissionTimestamp = conf.submissionTimestamp.apply().toString,
```

Comment:
Actually, putting parentheses after an object in Scala is the same as calling `apply`. The point of the syntactic sugar is to make the code less verbose, so let's use it.